File: frontend/src/app/models/Sums.ts
export class Sums {
  correct: number;
  wrong: number;
  totalTextAudioIndexes: number;

  constructor(correct: number, wrong: number, totalTextAudioIndexes: number) {
    this.correct = correct;
    this.wrong = wrong;
    this.totalTextAudioIndexes = totalTextAudioIndexes;
  }
}

File: backend/src/main/scala/models/Sums.scala
package models

import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}

case class Sums(
                 correct: Integer,
                 wrong: Integer,
                 totalTextAudios: Integer
               ) {
}

object Sums {
  implicit val encoder = deriveEncoder[Sums]
  implicit val decoder = deriveDecoder[Sums]
}
